plugins/flatpak/gs-flatpak.c

```diff
@@ -857,6 +858,18 @@ gs_flatpak_refresh_appstream_remote (GsFlatpak *self,
 	str = g_strdup_printf (_("Getting flatpak metadata for %s…"), remote_name);
 	gs_app_set_summary_missing (app_dl, str);
 	gs_plugin_status_update (self->plugin, app_dl, GS_PLUGIN_STATUS_DOWNLOADING);
+
+	if (!flatpak_installation_update_remote_sync (self->installation,
+						      remote_name,
+						      cancellable,
+						      &local_error)) {
+		g_debug ("Failed to update metadata for remote %s: %s\n",
+			 remote_name, local_error->message);
+		gs_flatpak_error_convert (&local_error);
+		g_propagate_error (error, g_pointer_steal (local_error));
```

Comment:
I think you meant `g_pointer_steal`. And it should be used with `&local_error` in this case.